docs/stopwatch.adoc

```diff
@@ -0,0 +1,22 @@
+:page-title: Measuring time with a Stopwatch
+:page-description: Extends JUnit Jupiter with `@Stopwatch` to measure elapsed time of a test method
+
+== Introduction
+
+Test shall be fast.
+To get a feeling how fast or slow a specific test is, the `@Stopwatch` annotation can be used.
+It will measure elapsed time and pass the result to the `https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/TestReporter.html[TestReporter]`, which will publish it.
```

Comment:
Hm, I'm not sure if this does what you think it does. Shouldn't it be: url[\`text\`] instead of \`url[text]\`

Reply:
It creates a link in the code format. And my IDEA ASCII DOC plugin tells me it works (the link is correctly clickable in the preview and opens the desired page)